server/main.js

```diff
@@ -16,7 +16,7 @@ var createApplication = function () {
 
 var startServer = function () {
 
-    var PORT = process.env.PORT || 1337;
+    var PORT = process.env.PORT || 4567;
```

Comment:
constraint of VM env working in. This one WIP... will revert port back to 1337 as last step, haven't been switching back and forth for now to save time on commits.